Makefile

```diff
@@ -4,9 +4,15 @@ build:
 	@jbuilder build @install
 
 clean:
+	@rm -f `find . -name 'bisect*.out'`
 	@jbuilder clean
 
+coverage: clean
```

Comment:
I'll remove this `clean` when jbuilder gets support for unconditionally running tests.

Reply:
If I am understanding correctly, jbuilder doesn't run tests if the project is already built?

Reply:
Ahh found the answer in bisect_ppx documentation.